server/routes/explore/api.py

```diff
@@ -84,7 +87,9 @@ def detect():
 #  - childEntityType: A type of child entity (optional)
 #
 @bp.route('/fulfill', methods=['POST'])
-@cache.cached(timeout=TIMEOUT, query_string=True)
+@cache.cached(timeout=TIMEOUT,
```

Comment:
Will doing cache for POST without the custom key cause trouble?  That is, should this PR go in before prod push?

Reply:
We should get this PR in the release if possible (cc @keyurva ). Without the custom key, contextHistory is ignored for cached /fulfill and /detect-and-fulfill requests. the query string itself is still used, just not the context history.